netbox/ipam/tables/ip.py

```diff
@@ -200,8 +200,9 @@ class PrefixTable(TenancyColumnsMixin, NetBoxTable):
         verbose_name=_('VLAN Group')
     )
     vlan = tables.Column(
+        accessor='vlan__vid',
```

Comment:
Can we leave the accessor as-is to retain the link to the object, and instead add `order_by=('vid', 'pk')` to only alter the ordering?